shared/templates/template_OVAL_package_installed

```diff
@@ -4,15 +4,25 @@
     <metadata>
       <title>Package %PKGNAME% Installed</title>
       <affected family="unix">
-        <platform>Debian 8</platform>
+        <platform>multi_platform_all</platform>
       </affected>
-      <description>The DEB package %PKGNAME% should be installed.</description>
+      <description>The {{{ pkg_system|upper }}} package %PKGNAME% should be installed.</description>
     </metadata>
     <criteria>
       <criterion comment="package %PKGNAME% is installed"
       test_ref="test_package_%PKGNAME%_installed" />
     </criteria>
   </definition>
+{{% if pkg_system == "rpm" %}}
+  <linux:rpminfo_test check="all" check_existence="all_exist"
+  id="test_package_%PKGNAME%_installed" version="1"
+  comment="package %PKGNAME% is installed">
+    <linux:object object_ref="obj_package_%PKGNAME%_installed" />
+  </linux:rpminfo_test>
+  <linux:rpminfo_object id="obj_package_%PKGNAME%_installed" version="1">
+    <linux:name>%PKGNAME%</linux:name>
+  </linux:rpminfo_object>
+{{% elif pkg_system == "dpkg" %}}
```

Comment:
Shouldn't this be `deb`?

Reply:
no. see https://en.wikipedia.org/wiki/Dpkg

deb is short for debian, dpkg is the packaging system

Reply:
Ok.